man/resolvectl.xml

```diff
@@ -531,6 +531,15 @@ _443._tcp.fedoraproject.org IN TLSA 0 0 1 19400be5b7a31fb733917700789d2f0a2471c0
         -- Matching type: SHA-256
 </programlisting>
     </example>
+
+    <example>
+      <title>Resolve local services</title>
+
+      <programlisting>$ resolvectl --type=PTR _googlecast._tcp.local
+                      $ resolvectl --type=SRV somenamefromprevious._googlecast._tcp.local
+                      $ resolvectl --type=A hostnamefromprevious.local
+      </programlisting>
```

Comment:
`programlisting` tag does not ignore the white spaces at the head. Please drop the white spaces at the beginning like the following:
```
        <programlisting>$ resolvectl --type=PTR _googlecast._tcp.local
$ resolvectl --type=SRV somenamefromprevious._googlecast._tcp.local
$ resolvectl --type=A hostnamefromprevious.local</programlisting>
```
You can test your man page is correctly formatted by running `man/man resolvectl` in your build directory.

Reply:
yes, I didn't notice the `programlisting`  format. Here's my testing man page After I drop the white spaces 
![image](https://user-images.githubusercontent.com/37774387/122769357-15c58580-d2d7-11eb-8fbd-dab211790c48.png)

Reply:
So... please drop the leading spaces.